plugins/10-setuppy_update.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+function run() {
+
+    local version_new="$1"
+    local git_root="$5"
+
+    local setuppy_path="${git_root}/setup.py"
+
+    if [ ! -e "${setuppy_path}" ]
+    then
+        echo "Could not find setup.py on the project root."
+        return 112
+    fi
+
+    tmpfile=$(mktemp)
+    sed "s/version[[:blank:]]*=[[:blank:]]*[\",'][0-9][0-9]*.[0-9][0-9]*.[0-9][0-9]*[\",'][[:blank:]]*,/version=\"${version_new}\",/" < "${git_root}/setup.py" > "${tmpfile}"
```

Comment:
Couple of queries around this regex:

* Are you sure you need the `,` in the character class `[\",']`?
* I think you need to escape your full stops (`.`), otherwise this will match any character.
* It might be worth documenting `[0-9][0-9]*` is for Mac compatibility.

Reply:
Hi Mark,

Thanks for your comments, and you are right about the `,` between the brackets, and escaping the dot, both were needed changes.

I also added a comment stating that I intentionally avoided using extended regular expressions for compatibility, so please let me know if you catch anything else that can be improved.

Thanks, Fernando